app/helpers/static_page_helper.rb

```diff
@@ -0,0 +1,2 @@
+module StaticPageHelper
```

Comment:
If you don't use this file just delete it

Reply:
deleted